jenkins/opensearch/integ-test.jenkinsfile

```diff
@@ -7,30 +7,30 @@
  * compatible open source license.
  */
 
-lib = library(identifier: 'jenkins@5.11.1', retriever: modernSCM([
+lib = library(identifier: 'jenkins@add-getNames', retriever: modernSCM([
```

Comment:
Why are we deviating from semver here?

Reply:
No was waiting for https://github.com/opensearch-project/opensearch-build-libraries/pull/361 to merge.

Reply:
Upgraded the version and git repo to right version now. 
Thanks!